go.mod

```diff
@@ -60,7 +60,7 @@ require (
 	google.golang.org/genai v1.37.0
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20251124214823-79d6a2a48846
 	google.golang.org/grpc v1.77.0
-	google.golang.org/protobuf v1.36.10
+	google.golang.org/protobuf v1.36.11
```

Comment:
i thought this is enough to fix the bug but didn't help. regardless, upgrading this in this PR proves that we need a workaround